### DISCUSSION TITLE
item de fidélité

### DISCUSSION THREAD

**Discussion by kekeishere** [Crownicles/Crownicles#1788] [Suggestions fermées]
on pourrait mettre un item ou genre chaque jour c'est comme un investissement si on se connecte tout les jour et que l'on fait une commande on a disont 10$ et a chaque fois que l on fait cette commande sa ajoute une certaine somme par exemple hein c est que un exemple **_10$ + X x nbr de jour connecté d'affilé en faisant la commande_** et comme sa si jamais pendant ne serait-ce que un seul jour on ne fait pas la commande le jour qui suit sa revient a 0 
une sorte de programme de fidélité avec un item 
par contre faut pas que l argent que l on rajoute en plus chaque soit trop élevé sinon c trop cheat

**Comment by Nargacaura** [comment]
L'idée n'est pas si mal que ça, mais.. est-ce que ça a vocation d'être un objet quotidien ou ça veut tenter de le remplacer s'il s'agit d'une commande spécifique à ce programme fidélité? 🤔

Si c'est une commande spécifique, je n'arrive pas à voir l'intérêt vu qu'on a des objets quotidiens d'argent qui peuvent en rapporter plus par jour via le guilddaily et rajouter une commande juste pour ça serait useless. Sinon, je pense qu'on pourrait mettre une fourchette 50-250 pour éviter d'atteindre des sommes équivalentes aux légendaires.

**Reply by kekeishere** [reply]
c un item genre bien casse tete a trouver

**Comment by BastLast** [comment]
Pour moi cette mécanique est deja remplie par les objets daily et les daily guild je trouve que cela n'apporte pas grand chose de rendre le truc progressif
